### PR TITLE
Adding HLDS-GIT to documentation index for distro

### PR DESCRIPTION
**Pull request by HLDS-GIT**
I'd like HLDS-GIT to be indexed and documented on ros.org.

### CODE REVIEW

File: kinetic/distribution.yaml [hunk 1]
```diff
@@ -3075,6 +3075,23 @@ repositories:
...
+    doc:
+      type: git
+      url: https://github.com/HLDS-GIT/hls_lfom_tof_driver.git
+      version: kinetic-devel
+    release:
+      packages:
+      - hls_lfom_tof_driver
+      tags:
+        release: release/kinetic/{package}/{version}
+      url: https://github.com/HLDS-GIT-release/hls-lfom-tof-driver-release.git
```
Comment (mikaelarguedas) [inline kinetic/distribution.yaml]: This repository doesnt seem to exist or to be public

File: kinetic/distribution.yaml [hunk 2]
```diff
@@ -3075,6 +3075,23 @@ repositories:
       url: https://github.com/ROBOTIS-GIT/hls_lfcd_lds_driver.git
       version: kinetic-devel
     status: maintained
+  hls-lfom-tof-driver:
+    doc:
+      type: git
+      url: https://github.com/HLDS-GIT/hls_lfom_tof_driver.git
+      version: kinetic-devel
```
Comment (mikaelarguedas) [inline kinetic/distribution.yaml]: this branch doesnt exist on https://github.com/HLDS-GIT/hls_lfom_tof_driver.git

**Comment by mikaelarguedas** [comment]
@HLDS-GIT To release a catkin package for the first time in ROS, you may find this tutorial useful: http://wiki.ros.org/bloom/Tutorials/FirstTimeRelease

**Comment by HLDS-GIT** [comment]
Adding HLDS-GIT to documentation index for distro

**Pull request merged**